tests/test_skyWcs.py

```diff
@@ -545,6 +545,15 @@ def testAgainstAstropyWcs(self):
             # Most projections only seem to agree to within 1e-4 in the round trip test
             self.assertSkyWcsAstropyWcsAlmostEqual(skyWcs=skyWcs, astropyWcs=astropyWcs, bbox=bbox)
 
+    def testStr(self):
+        """Test that we can get something coherent when printing a SkyWcs.
+        """
+        cdMatrix = makeCdMatrix(scale=self.scale)
+        skyWcs = makeSkyWcs(crpix=self.crpix, crval=self.crvalList[0], cdMatrix=cdMatrix)
+        self.assertIn(f"Sky Origin: {self.crvalList[0]}", str(skyWcs))
+        self.assertIn(f"Pixel Origin: {self.crpix}", str(skyWcs))
+        self.assertIn(f"Pixel Scale: ", str(skyWcs))
```

Comment:
Derive the expected value from `cdMatrix`?

Reply:
I don't understand this comment. You mean derive the pixel scale from it? It's potentially more complicated than just `cdMatrix` (e.g. optical distortions). I could put in a call to `getPixelScale()` here, but I decided I didn't want to be too specific in the unittest (e.g. units conversions).